docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -693,6 +773,130 @@ The ``FV3LAM_wflow.xml`` file runs the specific j-job scripts (``regional_workfl
    | run_post             | Run the post-processing tool (UPP)                         |
    +----------------------+------------------------------------------------------------+
 
+In addition to the baseline tasks described in :numref:`Table %s <WorkflowTasksTable>` above, users may choose to run some or all of the METplus verification tasks. These tasks are described in :numref:`Table %s <VXWorkflowTasksTable>` below. 
+
+.. _VXWorkflowTasksTable:
+
+.. table:: Verification (VX) workflow tasks in the SRW App
+
+   +-----------------------+------------------------------------------------------------+
+   | **Workflow Task**     | **Task Description**                                       |
+   +=======================+============================================================+
+   | GET_OBS_CCPA          | Retrieves and organizes hourly :term:`CCPA` data from NOAA |
+   |                       | HPSS. Can only be run if ``RUN_TASK_GET_OBS_CCPA="TRUE"``. |
+   +-----------------------+------------------------------------------------------------+
+   | GET_OBS_NDAS          | Retrieves and organizes hourly :term:`NDAS` data from NOAA |
+   |                       | HPSS. Can only be run if ``RUN_TASK_GET_OBS_NDAS="TRUE"``. |
+   +-----------------------+------------------------------------------------------------+
+   | GET_OBS_MRMS          | Retrieves and organizes hourly :term:`MRMS` composite      |
+   |                       | reflectivity and :term:`echo top` data from NOAA HPSS. Can |
+   |                       | only be run if ``RUN_TASK_GET_OBS_MRMS="TRUE"``.           |
+   +-----------------------+------------------------------------------------------------+
+   | VX_GRIDSTAT           | Runs METplus grid-to-grid verification for 1-h accumulated |
+   |                       | precipitation                                              |
+   +-----------------------+------------------------------------------------------------+
+   | VX_GRIDSTAT_REFC      | Runs METplus grid-to-grid verification for composite       |
+   |                       | reflectivity                                               |
+   +-----------------------+------------------------------------------------------------+
+   | VX_GRIDSTAT_RETOP     | Runs METplus grid-to-grid verification for :term:`echo top`|
+   +-----------------------+------------------------------------------------------------+
+   | VX_GRIDSTAT_03h       | Runs METplus grid-to-grid verification for 3-h accumulated |
```

Comment:
This is where you could state **h instead of 3/6/24 explicitly and stat it is available for 3/6/24.